ci/gitlab/.gitlab-ci.yml

```diff
@@ -434,6 +434,7 @@ build-osx-release:
   before_script:
     - brew uninstall -f aktualizr
     - brew untap advancedtelematic/otaconnect
+    - rm -rf /Users/gitlab/Library/Caches/Homebrew/aktualizr--git
```

Comment:
Slightly ugly that we have to both hardcode a path like that and use `rm -Rf`. Both of those make me uncomfortable. Are there other options?

Reply:
yeah, I don't like it too, I tried `brew cleanup -s` and `brew install -v -f --build-bottle aktualizr` ('f' option) but they do not do the job, looks like it's another brew issue according to what I read...

Reply:
Anyone else see any options here? Or if it's what we have to do, then so be it...

Reply:
This part of the path `/Users/gitlab/Library/Caches/Homebrew` can be obtained from `$(brew --cache)`, let me see if I can find anything for `aktualizr--git`

Reply:
> This part of the path `/Users/gitlab/Library/Caches/Homebrew` can be obtained from `$(brew --cache)`

Ah, interesting. The `rm` is still scary, but maybe less so.

> let me see if I can find anything for `aktualizr--git`

That's the part I'm least worried about. :)

Reply:
> Ah, interesting. The rm is still scary, but maybe less so.

the maximum what can be removed is the stuff that `gitlab` user is permitted to remove without sudo and it's not system critical stuff so the follow-up brew command should restore its required state.

Reply:
Fair point. And if the `aktualizr--git` part is hardcoded we can be fairly sure it's not going to do anything too dangerous.